gensrc/thrift/Descriptors.thrift

```diff
@@ -200,100 +175,6 @@ struct TColumn {
     21: optional Types.TTypeDesc type_desc         
 }
 
-struct TOlapTableIndexTablets {
```

Comment:
Is it possible here to keep the definition of the class from moving？

Reply:
If dont move class def, exprs and descriptors will include each other, and failed in compile.
because TExprNode.TDictQueryExpr use some table info class define in descriptors.thrift.